src/main/java/org/jabref/logic/layout/format/AuthorAndsReplacer.java

```diff
@@ -21,24 +21,18 @@ public String format(String fieldText) {
         String s;
 
         switch (authors.length) {
-        case 1:
-            //Does nothing;
-            s = authors[0];
-            break;
-        case 2:
-            s = authors[0] + " & " + authors[1];
-            break;
-        default:
-            int i;
-            int x = authors.length;
-            StringBuilder sb = new StringBuilder();
-
-            for (i = 0; i < (x - 2); i++) {
-                sb.append(authors[i]).append("; ");
+            case 1 -> s = authors[0]; // just no action
```

Comment:
why not use the full power of the switch expressions and write `String s = switch (...)` ?

Reply:
IntelliJ did not recommend it ^^. Just submit an update here :)